source/ProtonPack/Serial.h

```diff
@@ -1589,15 +1620,16 @@ void handleWandCommand(uint8_t i_command, uint16_t i_value) {
     break;
 
     case W_SLIME_MODE:
-      // Slime mode
-      FIRING_MODE = SLIME;
-      playEffect(S_CLICK);
+      if(AUDIO_DEVICE == A_GPSTAR_AUDIO && FIRING_MODE == MESON) {
+        // Tell GPStar Audio we no longer need short audio.
+        audio.gpstarShortTrackOverload(true);
+      }
 
-      stopEffect(S_PACK_SLIME_TANK_LOOP);
-      stopEffect(S_STASIS_IDLE_LOOP);
-      stopEffect(S_MESON_IDLE_LOOP);
+      if(PACK_STATE == MODE_ON && FIRING_MODE != SLIME) {
+        stopEffect(S_PACK_SLIME_TANK_LOOP);
+        stopEffect(S_STASIS_IDLE_LOOP);
+        stopEffect(S_MESON_IDLE_LOOP);
 
-      if(PACK_STATE == MODE_ON && b_wand_on == true) {
         playEffect(S_PACK_SLIME_OPEN);
         playEffect(S_PACK_SLIME_TANK_LOOP, true, 0, true, 900);
 
```

Comment:
One thing that needs to be adjusted, is when leaving Slime mode and in Afterlife or Frozen Empire, the S_AFTERLIFE_PACK_IDLE and STARTUP need to fade back in to full volume again. It does it currently when leaving Slime mode to enter Stasis mode, however with the top dial able to go reverse when switching firing modes now,, the Proton stream mode does not have the adjustGainEffect code.

So basically, just need to detect if in Afterlife or Frozen Empire mode and are leaving Slime mode, to run:

        if((SYSTEM_YEAR == SYSTEM_AFTERLIFE || SYSTEM_YEAR == SYSTEM_FROZEN_EMPIRE)) {
          adjustGainEffect(S_AFTERLIFE_PACK_STARTUP, i_volume_effects, true, 100);
          adjustGainEffect(S_AFTERLIFE_PACK_IDLE_LOOP, i_volume_effects, true, 100);
        }

Reply:
Whoops! Can't believe I missed that while fixing the cyclotron reset stuff.